### PR TITLE
Updating README.md and adding Docs!

### PR DESCRIPTION
**Pull request by MykalMachon**
_See #14 and #12 for context on this PR_

## Summary of Changes
This Pull Request will be for changes to and creation of documentation as per #14. 

In essence, We're looking to create user friendly documentation that encourages and enables CSA members that are new to open source to contribute to this repo!

Things added in this PR: 
- An updated `README.md` with updated information introducing people to the project, our goals, and where they can help out!  👋 
- An updated `CONTRIBUTING.md` with updated local development guidelines, info on how to clone the repo, and links to resources 📖 
- A `README.md` file in both `/content/` and `/src/` explaining the core concepts of each folder and linking off to more resources 📂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+### Goals
+
+This repository has a few key goals:
+
+1. Maintain the UFV CSA website with up-to-date and relevant content for CSA members.
+2. Provide CSA members with an opportunity to work with and learn valuable workplace skills including:
+   - Team Communication: How to effectively work with others on a real-world project.
+   - Git / Github Workflows: How to manage code-bases using git / github
+   - Realworld Web Technologies: How to work with modern web technologies like Gatsby.js, React, and Node.
+3. Add more goals / re-arrange
```
Comment (MykalMachon) [inline README.md]: @MJJBennett Was wondering if you had any goals in mind for this project or if you agreed with what I've listed here so far? Thanks!

File: README.md [hunk 2]
```diff
@@ -1,5 +1,81 @@
...
+2. Provide CSA members with an opportunity to work with and learn valuable workplace skills including:
+   - Team Communication: How to effectively work with others on a real-world project.
+   - Git / Github Workflows: How to manage code-bases using git / github
+   - Realworld Web Technologies: How to work with modern web technologies like Gatsby.js, React, and Node.
+
+## Contributing
+
+**The website is currently in development!** If you have any ideas involving:
+
+1. **Content:** Found a typo on the website? Have an even't you'd like to share with other members of the CSA? Have other content ideas? We'd love for you to make a contribution!
```
Comment (MJJBennett) [inline README.md]: Speaking of typos ( 😉 ) `even't` should probably be `event` here

File: README.md [hunk 3]
```diff
@@ -1,5 +1,81 @@
...
+2. **Styling:** Don't like the way the site looks? Found a visual bug? We're open to suggestions and are planning on re-designing the site soon so let us know!
+3. **Use-cases & Design:** Do you have an idea for a useful new feature? Do you feel like a feature on the site could be implemented better? Let's work on it!
+4. **Pretty Much Anything Else**: We're open to general suggestions as well.
+
+If anything listed above sounds like something you'd like to do, **We'd love for you to contribute and become part of the CSA's Open Source Community**.
+
+We've tried to design this repository to be as accessible to all as possible. Whether you've never commited to open source before, or you're an active member of the community
```
Comment (MJJBennett) [inline README.md]: WIP sentence?

Comment (MykalMachon) [reply README.md]: Ha, nope must have just got distracted. Thanks for pointing that out.

**Comment by MykalMachon** [comment]
@MJJBennett saw you approved this PR but didn't merge it, was there a specific reason for that! Would love to work it out ASAP. Thanks!

**Comment by MJJBennett** [comment]
@MykalMachon Hey, super sorry about that. I've been thinking and trying to come up with a good solution to the hosting problem, and thinking about the issue of how to approach student involvement.

But, this is perfectly fine as a pull request, so I'll merge it now.

**Comment by MykalMachon** [comment]
@MJJBennett No worries at all! I've been trying to plot out a stable solution for hosting as well! 

If there is anything I can help with regarding either of these issues be sure to let me know! Thanks!

**Comment by MJJBennett** [comment]
@MykalMachon No worries on that front right now, the website currently exists which is a bump up already. Once midterm season is over I think it'll be easier to look into this.

**Comment by MykalMachon** [comment]
@MJJBennett agreed :+1:

**Pull request merged**